test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -833,6 +833,58 @@ namespace StackTest {
             $LASTEXITCODE | Should -Be $ExitCodeBadCommandLineParameter
         }
     }
+
+    Context "Startup banner text tests" -Tag Slow {
+        BeforeAll {
+            $waitSecs = 3
+            $outputPath = Join-Path $env:temp "StartupBanner-${Pid}.txt"
+
+            # Not testing update notification banner text here
+            $origUpdateCheckVal = $env:POWERSHELL_UPDATECHECK
+            $env:POWERSHELL_UPDATECHECK = "Off"
+
+            $spArgs = @{
+                FilePath = $powershell
+                ArgumentList = @("-NoProfile")
+                RedirectStandardOutput = $outputPath
```

Comment:
Can we use `RedirectStandardInput` and send `Exit` command so that avoid Start-Sleep?

Reply:
I think so.  I'll still need to use Start-Sleep in a `while` loop to wait for `!$p.HasExited` but this would be more robust.  Thanks.

Reply:
Check that, I can use `$p.WaitForExit(5000)`?

Reply:
Yes, you can. Or we could use our helper:
```
Wait-UntilTrue -sb { $process.HasExited } -TimeoutInMilliseconds 5000 -IntervalInMilliseconds 1000 | Should -BeTrue
```